<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRoomDlg</class>
 <widget class="QDialog" name="AddRoomDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>幼圆</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>添加房间</string>
  </property>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>房间编号（1-100）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>房间名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>房间优先级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRoomIdx">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRoomName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonPrioLow">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>低</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonPrioMid">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>中</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonPrioHigh">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>高</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>lineEditRoomName</tabstop>
  <tabstop>lineEditRoomIdx</tabstop>
  <tabstop>radioButtonPrioLow</tabstop>
  <tabstop>radioButtonPrioMid</tabstop>
  <tabstop>radioButtonPrioHigh</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
